www/i18n/en.json

```diff
@@ -307,27 +307,28 @@
           "android-10": "Please select 'Allow all the time'",
           "android-11": "On the app settings page, choose the 'Location' permission and set it to 'Allow all the time'",
           "android-gte-12": "On the app settings page, choose the 'Location' permission and set it to 'Allow all the time' and 'Precise'",
-          "ios-lt-13": "Please select 'Always allow'",
-          "ios-gte-13": "On the app settings page, please select 'Always' and 'Precise' and return here to continue"
+          "ios-lt-13": "Select 'Always Allow' to enable location tracking",
+          "ios-13-13.3": "Select 'Always' location permissions on the next screen to enable location tracking, then return here to continue",
+          "ios-gte-13.4": "Select 'Precise' and 'Allow While Using App' on the next screen, then 'Change to Always Allow' to enable location tracking"
         }
       },
       "overall-fitness-name-android": "Physical activity",
       "overall-fitness-name-ios": "Motion and Fitness",
-      "overall-fitness-description": "The fitness sensors distinguish between walking, bicycling and motorized modes.  We use this data in order to separate the parts of multi-modal travel such as transit. We also use it to as a cross-check potentially spurious trips - if the location sensor jumps across town but the fitness sensor is stationary, we can guess that the trip was invalid.",
+      "overall-fitness-description": "The fitness sensors distinguish between walking, bicycling and motorized modes. We use this data in order to separate the parts of multi-modal travel such as transit. We also use it to as a cross-check potentially spurious trips - if the location sensor jumps across town but the fitness sensor is stationary, we can guess that the trip was invalid.",
       "fitnessperms": {
         "name": "Fitness Permission",
         "description": {
           "android": "Please allow.",
```

Comment:
nit, future fix: if we are going to list out the details, we should do so on android as well.